tools/deb/istio-iptables.sh

```diff
@@ -58,8 +58,8 @@ function usage() {
 }
 
 function dump {
-    iptables-save
-    ip6tables-save
+    iptables-save -w
```

Comment:
This doesn't exist in either Fedora or RHEL for me. What is it supposed to do?

```
 iptables-save -w
iptables-save: invalid option -- 'w'
Look at manual page `iptables-save.8' for more information.
```

Reply:
Thanks @brian-avery! I noticed that in my other PR in Maistra/istio, but forgot to fix these two. {{iptables-restore}} supports the {{-w}} flag, but {{iptables-save}} does not.